src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -181,6 +178,8 @@ internal void BindToDataSource(MirrorData mirrorData)
             {
                 RefreshExpandedDisplay(delegate { BeginViewSizeTransition(ComputeLargeContentSize()); });
             }
+
+            IsDataBound = true;
```

Comment:
The purpose of `IsDataBound` becomes weird now after the change, I don't mind keeping it this way just to keep the change small, but we should follow this up with a subsequent task (also to consider renaming of `RunOnSchedulerSync` since it is essentially an asynchronous operation).